browser/src/slideshow/PresenterConsole.js

```diff
@@ -257,6 +304,18 @@ class PresenterConsole {
 		}
 	}
 
+	_onTransitionStart(e) {
+		if (!this._proxyPresenter) {
+			return;
+		}
+
+		this._currentIndex = e.slide;
+
+		let next =
+			this._proxyPresenter.document.querySelector('#next-presentation');
+		this.drawNext(next);
```

Comment:
Shall you limit `_onTransitionStart` to prefecthing ? It's a bit odd to see the next next slide when the next one has not yet been displayed.

Reply:
Changes done. Thanks